src/Umbraco.Web/Redirects/RedirectUrlSearchResults.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using Umbraco.Core.Models;
+
+namespace Umbraco.Web.Redirects
+{
+    public class RedirectUrlSearchResult
+    {
+        public string StatusMessage { get; set; }
```

Comment:
normally we'd attribute these models with things like `[DataMember(Name = "statusMessage")]` to force camelCase them so that the JS naming conventions can be correct.